From 9f44a74f93020ed27fb4279ab0d4bb2535089f03 Mon Sep 17 00:00:00 2001
From: Minjae Kim <nate.kim@lge.com>
Date: Thu, 28 Mar 2019 10:15:26 +0900
Subject: [PATCH 1/8] Simplify wayland-scanner lookup

Don't use pkg-config to lookup the path of a binary that's in the path.

Alternatively we'd have to prefix the path returned by pkg-config with
PKG_CONFIG_SYSROOT_DIR.

Upstream-Status: Pending
---
 configure.ac | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index fd346c8..0bb6db1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1669,12 +1669,7 @@ if test "x$with_platforms" = xauto; then
     with_platforms=$with_egl_platforms
 fi
 
-PKG_CHECK_MODULES([WAYLAND_SCANNER], [wayland-scanner],
-        WAYLAND_SCANNER=`$PKG_CONFIG --variable=wayland_scanner wayland-scanner`,
-        WAYLAND_SCANNER='')
-if test "x$WAYLAND_SCANNER" = x; then
-    AC_PATH_PROG([WAYLAND_SCANNER], [wayland-scanner], [:])
-fi
+AC_PATH_PROG([WAYLAND_SCANNER], [wayland-scanner])
 
 # Do per platform setups and checks
 platforms=`IFS=', '; echo $with_platforms`
-- 
2.17.1

